react-hot-button-component/my-app/src/HotButton.js

```diff
@@ -0,0 +1,61 @@
+import { useState } from 'react';
+
+export default function HotButton() {
+
+  const [clickCount, setClickCount] = useState(0);
+  const [buttonStyle, setButtonStyle] = useState({
```

Comment:
This can be computed from `clickCount` so it should not be state. Just do all those calculations in the main body of the function and set the buttonStyle variable there.